file_handling.rb

```diff
@@ -0,0 +1,21 @@
+#Read from a CSV file, multiple columns (minimum 2), and then write back to the CSV file.
+
+cs = File.open("addresses.csv",'r+')
+lines = cs.readlines
+for line in lines
+  puts line
+end
+new = [ "Ali" , "Noor" , "NA" , "NA" , "NA" ] 
```

Comment:
Use `%w` syntax for array of strings
https://github.com/github/rubocop-github/blob/master/STYLEGUIDE.md#collections